### PR TITLE
Localization properties

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds a localiztion property table to track skymap properties for filtering.

### CODE REVIEW

File: skyportal/handlers/api/gcn.py [hunk 1]
```diff
@@ -900,6 +1015,69 @@ def add_tiles(localization_id):
...
+            for text in tags_list
+        ]
+        session.add_all(tags)
+
+        session.commit()
+        return log(f"Generated properties for localization {localization_id}")
+    except Exception as e:
+        return log(
+            f"Unable to generate properties for localization {localization_id}: {e}"
+        )
```
Comment (guynir42) [inline skyportal/handlers/api/gcn.py:1076]: Return log seems kind of strange...

File: skyportal/handlers/api/gcn.py [hunk 2]
```diff
@@ -900,6 +1015,69 @@ def add_tiles(localization_id):
...
+    finally:
+        Session.remove()
```
Comment (guynir42) [inline skyportal/handlers/api/gcn.py:1078]: Why `remove`?

File: skyportal/models/localization.py [hunk 3]
```diff
@@ -183,3 +204,61 @@ class LocalizationTile(Base):
...
+    )
+
+    localization_id = sa.Column(
+        sa.ForeignKey('localizations.id', ondelete="CASCADE"),
+        primary_key=True,
+        index=True,
+        doc='localization ID',
+    )
+
+    data = sa.Column(JSONB, doc="Localization properties in JSON format.")
```
Comment (guynir42) [inline skyportal/models/localization.py:235]: If you don't index on this, that query for getting all the keys for all the data could be long.

File: skyportal/models/localization.py [hunk 4]
```diff
@@ -183,3 +204,61 @@ class LocalizationTile(Base):
...
+    )
+
+    localization_id = sa.Column(
+        sa.ForeignKey('localizations.id', ondelete="CASCADE"),
+        primary_key=True,
+        index=True,
+        doc='localization ID',
+    )
+
+    text = sa.Column(sa.Unicode, nullable=False)
```
Comment (guynir42) [inline skyportal/models/localization.py:264]: Probably want to index this, too.

**Review by guynir42 — APPROVED** [review]
Some minor questions/suggestions

File: skyportal/handlers/api/gcn.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/gcn.py]: agreed.

File: skyportal/handlers/api/gcn.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/handlers/api/gcn.py:1074]: That's what most of our IOloop sessions call.

File: skyportal/models/localization.py (hunk 3 above)
Comment (mcoughlin) [reply skyportal/models/localization.py]: makes sense.

File: skyportal/models/localization.py (hunk 4 above)
Comment (mcoughlin) [reply skyportal/models/localization.py]: makes sense.

**Pull request merged**